src/opera/util/metfile.py

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright 2022, by the California Institute of Technology.
+# ALL RIGHTS RESERVED.
+# United States Government sponsorship acknowledged.
+# Any commercial use must be negotiated with the Office of Technology Transfer
+# at the California Institute of Technology.
+# This software may be subject to U.S. export control laws and regulations.
+# By accepting this document, the user agrees to comply with all applicable
+# U.S. export laws and regulations. User has the responsibility to obtain
+# export licenses, or other export authority as may be required, before
+# exporting such information to foreign countries or providing access to
+# foreign persons.
+#
+
+"""
+=========
+metfile.py
+=========
+
+Creates .met file for the OPERA PGEs.
+
+This module is adapted for OPERA from the NISAR PGE R2.0.0 util/metfile.py
+Original Author: Alice Stanboli
+Adapted By: Jim Hofman
+
+"""
+
+
+import json
+import jsonschema
+import os
+
+
+class MetFile:
+    """Class read and write .met files."""
+
+    def __init__(self, met_file_name, met_dict={}):
```

Comment:
It's bad practice in python to use a mutable object (in this case a dictionary) as a default argument for `met_dict`. Have `None` be the default value and if `None` is provided, initialize `met_dict` to an empty dictionary in the body of ``__init__``

Reply:
Done